src/v2/Apps/Artist/Components/ArtistConsignButton.tsx

```diff
@@ -15,14 +18,35 @@ import {
   color,
 } from "@artsy/palette"
 import { AnalyticsSchema, useTracking } from "v2/Artsy"
+import { useFragment } from "relay-hooks"
 
 export interface ArtistConsignButtonProps {
-  artist: ArtistConsignButton_artist
+  artist: ArtistConsignButton_artist$key
```

Comment:
I wish this wasn't so, but /shrug

Reply:
👁️  👁️ 

Curious, they mention it in the docs but not *why*? It's linted at least?

Reply:
I don't remember the reason, but I kind like it. I remember it made things easier when I has to pass relay data from one comp to another, using $key made it a bit cleaner. I know I'm not helping much here, but trust it, with some usage I feel you will agree too.